docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_matview.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ALTER MATERIALIZED VIEW statement [YSQL]
+headerTitle: ALTER MATERIALIZED VIEW
+linkTitle: ALTER MATERIALIZED VIEW
+description: Use the `ALTER MATERIALIZED VIEW` statement to change the definition of a materialized view.
+menu:
+  preview:
+    identifier: ddl_alter_matview
+    parent: statements
+aliases:
+  - /preview/api/ysql/commands/ddl_alter_matview/
+type: docs
+---
+
+## Synopsis
+
+Use the `ALTER MATERIALIZED VIEW` statement to change the definition of a materialized view.
+
+## Syntax
+
+{{%ebnf%}}
+  alter_materialized_view,
+  alter_materialized_view_action,
+{{%/ebnf%}}
+
+## Semantics
+
+### *alter_materialized_view_action*
+
+Specify one of the following actions.
+
+#### RENAME TO *new_name*
+
+Rename the materialized view to the specified name.
+
+{{< note title="Note" >}}
+
+Renaming a materialized view is a non-blocking metadata change operation.
+
+{{< /note >}}
+
+
+#### SET TABLESPACE *tablespace_name*
+
+Asynchronously change the tablespace of an existing materialized view. 
+The tablespace change will immediately reflect in the config of the materialized view, however the tablet move by the load balancer happens in the background. 
+While the load balancer is performing the move it is perfectly safe from a correctness perspective to do reads and writes, however some query optimization that happens based on the data location may be off while data is being moved.
+
+{{< note title="Note" >}}
+
+This is a BETA feature, and we do not recommend running this command on a production database. You will see a warning when running the command, but you can ignore it. Here is an example below:
+
```

Comment:
This is no longer a beta feature, so we can remove this warning.

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/20562/commits/b6623939e056e04feba0e89ee7f5fa8a2ebd3a09